src/main.cpp

```diff
@@ -32,7 +32,9 @@ int main(int argc, char** argv)
 	
 	// Create the game and run the main loop
 	FGame game(engine);
+	game.loadCharacterType(FCharacterType::DEBUG,
+	"images/business.png"); // or all the available filenames should be saved as in string (without loading images) and just use FCharacterType?
```

Comment:
Copying-Pasing the comment:
// or all the available filenames should be saved as in string (without loading images) and just use FCharacterType?